sub-packages/bionemo-esm2/src/bionemo/esm2/scripts/train_esm2.py

```diff
@@ -339,8 +316,20 @@ def main(
             filename="{epoch}-{step}-{consumed_samples}",
```

Comment:
This isn't part of your changes, but reviewing the file, I think `filename` here is missing an f-string prefix?

Reply:
Oh nevermind, ptl doing some magic under the hood, ignore!: https://lightning.ai/docs/pytorch/stable/api/lightning.pytorch.callbacks.ModelCheckpoint.html